trestle/core/commands/author/versioning/template_versioning.py

```diff
@@ -72,16 +72,28 @@ def update_template_folder_structure(task_path: Path) -> None:
                 filter(lambda p: p.is_file(), file_utils.iterdir_without_hidden_files(task_path))
             )
 
+            version_regex = r'[0-9]+.[0-9]+.[0-9]+'
+            pattern = re.compile(version_regex)
```

Comment:
I think this pattern repeats, so it would be good as a const.  I think there are related regexes and they should also be named consts.